app/routes/endemics/biosecurity.js

```diff
@@ -172,7 +172,7 @@ const postHandler = {
       setEndemicsClaim(request, biosecurityKey, typeOfLivestock === pigs ? { biosecurity, assessmentPercentage } : biosecurity)
 
       if (biosecurity === 'no') {
-        raiseInvalidDataEvent(request, biosecurityKey, `Value ${biosecurity} is not equal to required value yes`)
+        await raiseInvalidDataEvent(request, biosecurityKey, `Value ${biosecurity} is not equal to required value yes`)
```

Comment:
Just wondering, do we want to await all these? They have no bearing on the response to the user, but will now delay it slightly.

Reply:
I'll remove these additions for now, as adding the await doesn't really give us anything